doc/languages-frameworks/index.xml

```diff
@@ -7,8 +7,9 @@
  </para>
  <xi:include href="agda.section.xml" />
  <xi:include href="android.section.xml" />
+ <xi:include href="bower.section.xml" />
```

Comment:
This should still be alphabetical.

Reply:
Yep will fix in a bit.

I am squashing the branch at the moment so will resolve this when i finish that

Reply:
Ok finally updated this one.